### ISSUE TITLE
Auto fetch Players and name and Id from data ba? 

### ISSUE THREAD

**Issue by Bruh-epic-yee** [Stoolbend/acc-toolkit#16]
Fetch players Id from excel?

**Comment by Bruh-epic-yee** [comment]
![image](https://github.com/Stoolbend/acc-toolkit/assets/1040992/a49eaff5-0c50-467b-8a72-88ae5c84da07)

Hi Conor, 

We have a small data with names and Id, 
Is it possible acc toolkit can fetch Id from excel files automatically? 


Love your acc toolkit! 

Many thanks

**Comment by Stoolbend** [comment]
Thanks 😄
When you say about importing drivers - do you mean importing them to the saved driver function on the new version?
Once drivers are in the "saved drivers" list then they can be loaded into the entry lists/wherever else drivers might need to be used.

I think that should be fairly easy to pull off if the file format is something like CSV (which plenty of stuff like google sheets, excel can save to).

**Comment by Bruh-epic-yee** [comment]
Yeah, We have a few excels files with name and ids, is there anyway can be transferred / imported into “saved drivers”? 

We collect following from our races manually, we have a few excel files. 
I can combined into one big list, however, duplicates will be in data. 

Many thanks 🙏

**Comment by Stoolbend** [comment]
Sure thing - i'll have a look once i've finished a few other parts first.
I've removed the images just in case there are issues with peoples names & player IDs being public too - just in case you wonder where it's gone.
I'll try and aim for the import format to be something simple 👍

**Comment by Bruh-epic-yee** [comment]
apologise, with the expose of name
And id.

**Comment by Bruh-epic-yee** [comment]
Nice. Thank you 🙏

**Comment by Bruh-epic-yee** [comment]
Hi, Conor, 

Saved data fuction works so convenient!

DATA bank for players Id saved, will be available to anyone who use acc toolkit?
or can it be accessable to only person who uploaded the data?

many thanks

**Comment by Stoolbend** [comment]
> Saved data fuction works so convenient!
> DATA bank for players Id saved, will be available to anyone who use acc toolkit? or can it be accessable to only person who uploaded the data?

Saved drivers are stored inside your browser's "local storage", so are only useable on the device & browser you originally added them with. 👍
I will eventually add a method of exporting & importing the data though so if you run a league or something, you can share the saved driver list in a file.

**Comment by Bruh-epic-yee** [comment]
Nice. 
Looking forward to exporting and importing functions, 
more than happy to help to test new features. 

Many thanks
